tools/c7n_azure/c7n_azure/resources/kusto.py

```diff
@@ -0,0 +1,13 @@
+from c7n_azure.resources.arm import ArmResourceManager
+from c7n_azure.provider import resources
+
+
+@resources.register('kusto')
+class Kusto(ArmResourceManager):
+    class resource_type(ArmResourceManager.resource_type):
+        doc_groups = ['Kusto']
```

Comment:
this should go into either the Analytics or Monitoring doc groups.

Reply:
changed it to `Analytics` because [this](https://learn.microsoft.com/en-us/connectors/kusto/) docs tells that it "is a log analytics cloud platform"